ckan/config/middleware/flask_app.py

```diff
@@ -41,6 +41,31 @@
 log = logging.getLogger(__name__)
 
 
+class CKANBabel(Babel):
+    def __init__(self, *pargs, **kwargs):
+        super(CKANBabel, self).__init__(*pargs, **kwargs)
+        self.__i18n_path_idx = 0
```

Comment:
double underscore names cause name mangling, is this necessary?

Reply:
Actually - no. This class does not suppose to be an ancestor for any further extending. I just wanted to notify, that this property not just internal value, but rather very-very internal pointer that does not assume not only external access but even internal(except for domain property)